en/python_introduction/README.md

```diff
@@ -793,7 +793,11 @@ Hi there!
 How are you?
 ```
 
-That was easy! Let's build our first function with parameters. We will use the previous example – a function that says 'hi' to the person running it – with a name:
+If you see an error instead, don't panic! Python is trying to help you find the source of the problem.
+If you get a **NameError**, that probably means you typed something wrong, so you should check that you used the same name when creating the function with `def hi():` and when calling it with `hi()`.
+If you get an **IndentationError**, check you don't have any extra whitespace at the start of the line: python wants all the code inside in your function to be neatly aligned.
+
```

Comment:
Yeah, that makes sense. I've reformatted it as a note with a list. What do you think of the latest commit?